### PR TITLE
Add prodedure/troubleshooting topic for CNI version-related breaking …

### PR DESCRIPTION
**Pull request by rolfedh**
Add prodedure/troubleshooting topic for CNI version-related breaking change https://github.com/kubernetes/kubernetes/issues/109686

Also fixes #33240

IMPORTANT
Please use the PR for the Blog post, https://github.com/kubernetes/website/pull/33240, for all discussions, not this one. That way, you're seeing the latest content. I'll migrate all changes from that PR to this one (which has nearly the same content) once it has been reviewed and approved.

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/updrading-and-configuring-cni-plugins.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+broken, see the following [Troubleshooting](#troubleshooting) section.
+
+## Explanation
+
+CNI enables Kubernetes 1.24 networking. The given container runtimes support the
+most current versions of CNI, v1.0.0 and later. For CNI to work, you must use a
+CNI plugin, and its configuration file must specify the CNI version for which
+the plugin was compiled.
+
+If the version of the plugin is not specified, CNI v1.0.0 determines that the
```
Comment (aojea) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/updrading-and-configuring-cni-plugins.md:61]: @squeed is this correct?

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/updrading-and-configuring-cni-plugins.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+You must safely drain and cordon a node before upgrading the CNI plugins on the
+node to a current version. Draining is required because the currently running
+pods have a networking setup based on the older CNI plugins. To correctly tear
+down the network on the running pods, you should leave the older version of the
+CNI Plugins in place to perform the cordoning operation correctly.
+
+## An example containerd configuration file
+
+The following example shows a configuration for `containerd` runtime v1.6.x,
```
Comment (aojea) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/updrading-and-configuring-cni-plugins.md:75]: I think that 1.0.0 is backwards compatible and 0.4.0 plugins will work as well

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/updrading-and-configuring-cni-plugins.md (hunk 1 above)
Comment (rolfedh) [reply content/en/docs/tasks/administer-cluster/migrating-from-dockershim/updrading-and-configuring-cni-plugins.md:61]: @aojea @squeed 
Please use the PR for the Blog https://github.com/kubernetes/website/pull/33240 for all discussions, not this PR. That way, you're seeing the latest content. I'll migrate all changes from that PR to this one (which has nearly the same content) once it has been reviewed and approved.

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/updrading-and-configuring-cni-plugins.md (hunk 2 above)
Comment (rolfedh) [reply content/en/docs/tasks/administer-cluster/migrating-from-dockershim/updrading-and-configuring-cni-plugins.md:75]: @aojea
Please use the PR for the Blog https://github.com/kubernetes/website/pull/33240 for all discussions, not this PR. That way, you're seeing the latest content. I'll migrate all changes from that PR to this one (which has nearly the same content) once it has been reviewed and approved.

Comment (aojea) [reply content/en/docs/tasks/administer-cluster/migrating-from-dockershim/updrading-and-configuring-cni-plugins.md:75]: Yep, sorry

**Comment by rolfedh** [comment]
Issue resolved by updating the software and updating the release note. This PR is no longer needed.
Context: https://kubernetes.slack.com/archives/C0156PJ62RE/p1651098266321399?thread_ts=1650379911.716169&cid=C0156PJ62RE

**Pull request closed without merging**